BUILD.gn

```diff
@@ -495,6 +495,7 @@ source_set("electron_lib") {
       "LocalAuthentication.framework",
       "QuartzCore.framework",
       "Quartz.framework",
+      "QuickLookThumbnailing.framework",
```

Comment:
shouldn't the framework be weakly linked?

Reply:
@miniak where would you suggest we add it? When i tried initially it wouldnt link unless i added it here

Reply:
You can use `weak_frameworks`.